src/script/engine.zig

```diff
@@ -358,6 +363,82 @@ pub const Engine = struct {
         try self.stack.pushByteArray(value);
     }
 
+    /// OP_NIP: Removes the second-to-top stack item
+    ///
+    /// # Returns
+    /// /// - "EngineError.StackUnderflow": if initial stack length < 2
+    fn opNip(self: *Engine) !void {
+        if (self.stack.len() < 2) {
+            return error.StackUnderflow;
+        }
```

Comment:
you don't need to check if you are using `try` later on. Or you don't need to use `try` and can use `catch unreachable instead`